src/glue/job.ts

```diff
@@ -52,45 +52,174 @@ export interface ContinuousLoggingProps {
+   * 
+   * @see [AWS::Glue::Job NotificationProperty](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-glue-job-notificationproperty.html)
+   */
   readonly notifyDelayAfter?: Duration;
+  /**
+   * The name or Amazon Resource Name (ARN) of the IAM role associated with this job.
+   */
   readonly role?: IRole;
+  /**
+   * The Security Configuration object to be applied to the Job 
+   */
   readonly securityConfiguration?: SecurityConfiguration;
+  /**
+   * The job timeout in minutes. This is the maximum time that a job run can consume resources before it is terminated and enters TIMEOUT status. The default is 2,880 minutes (48 hours).
+   * 
+   * @see [AWS::Glue::Job](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-job.html#cfn-glue-job-timeout)
+   */
   readonly timeout?: Duration;
+  /**
+   * The number of worker available the Job 
+   */
   readonly workerCount?: number;
+  /**
+   * The type of predefined worker that is allocated when a job runs. Accepts a value of Standard, G.1X, or G.2X.
+   *    - For the Standard worker type, each worker provides 4 vCPU, 16 GB of memory and a 50GB disk, and 2 executors per worker.
+   *    - For the G.1X worker type, each worker maps to 1 DPU (4 vCPU, 16 GB of memory, 64 GB disk), and provides 1 executor per worker. We recommend this worker type for memory-intensive jobs.
+   *    - For the G.2X worker type, each worker maps to 2 DPU (8 vCPU, 32 GB of memory, 128 GB disk), and provides 1 executor per worker. We recommend this worker type for memory-intensive jobs.
+   * 
+   * @see [AWS::Glue::Job](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-job.html#cfn-glue-job-workertype)
+   */
   readonly workerType?: WorkerType;
 }
 
+/**
+ * Creates a Glue Job 
+ * 
+ * @see [AWS::Glue::Job](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-glue-job.html)
+ */
+
 export class Job extends Resource {
   // Internal properties
   private readonly _arguments: {[key: string]: string} = {};
   private readonly _connections: Connection[] = [];
 
   // Input properties
+  /**
+   * {@link JobProps.allocatedCapacity }
+   */
   public readonly allocatedCapacity?: number;
+  /**
+   * {@link JobProps.connections}
+   */
   public readonly connections?: Connection[];
-  public readonly continuousLogging?: ContinuousLoggingProps;
-  public readonly description?: string;
-  public readonly executable: JobExecutable;
-  public readonly maxCapacity?: number;
-  public readonly maxConcurrentRuns?: number;
-  public readonly maxRetries?: number;
-  public readonly name?: string;
-  public readonly notifyDelayAfter?: Duration;
-  public readonly securityConfiguration?: SecurityConfiguration;
-  public readonly timeout?: Duration;
-  public readonly workerCount?: number;
-  public readonly workerType?: WorkerType;
+  /**
```

Comment:
Indentation looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now